root/static/scripts/release-editor/fields.js

```diff
@@ -293,6 +294,10 @@ class Track {
         return hasVariousArtists(this.artistCredit());
     }
 
+    hasFeatOnTitle() {
+        return featRegex.test(this.name().replace(/[\[\]()]/g," "));
+    }
+
```

Comment:
It should probably use `bracketPairs` which has some more bracket variants.

Reply:
Isn't that the same two but with and without spaces?

Reply:
No, these are Japanese variants.

Reply:
Oooooh! Ok :)

Reply:
So, bracketPairs is annoying because they're, well paired. I just copied the Japanese variants here too instead (because the pairs are only useful to check they are balanced and whatnot, which we don't care about here).

Reply:
So just flatten them, see below <https://github.com/metabrainz/musicbrainz-server/pull/1546#discussion_r485618374>.